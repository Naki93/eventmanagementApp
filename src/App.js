
import './App.css';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import RegistrationForm from './components/RegistrationForm';
import LoginForm from './components/LoginForm';
import HomePage from './components/HomePage';
import AdminDashboard from './components/AdminDashboard';
import UserDashboard from './components/UserDashboard';

function App() {
  return (
    <div className="App">
      <Router>
      <Routes>
      <Route path="/" element={<HomePage />} />
        <Route path="/register" element={<RegistrationForm />} />
        <Route path="/login" element={<LoginForm />} />
        
        <Route path="/admin-dashboard" element={<AdminDashboard />} />
        <Route path="/user-dashboard" element={<UserDashboard />} />
       
      </Routes>
    </Router>
    </div>
  );
}

export default App;
