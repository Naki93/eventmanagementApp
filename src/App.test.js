
// import React from 'react';
// import { render } from '@testing-library/react';
// import App from './App';


// test('renders App component correctly', () => {
//   const { container } = render(<App />);
//   expect(container).toMatchSnapshot();
// });

// import React from 'react';
// import { render } from '@testing-library/react';
// import { MemoryRouter, Routes, Route , Router} from 'react-router-dom';
// import App from './App';

// test('renders App component correctly', () => {
//   const { container } = render(<App />);

//   expect(container).toMatchSnapshot();
// });
import React from 'react';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import { render } from '@testing-library/react';
import App from './App';

test('renders App component correctly', () => {
  const { container } = render(
    
        <App />
  );

  expect(container).toMatchSnapshot();
});

