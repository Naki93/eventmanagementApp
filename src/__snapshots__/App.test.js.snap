// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component correctly 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="home-page"
    >
      <nav
        class="navbar navbar-dark bg-dark"
      >
        <div
          class="container"
        >
          <a
            class="navbar-brand logo"
            href="#"
          >
            Eventify
          </a>
          <span
            class=""
          >
            Your Go-To Event Platform
          </span>
          <div
            class="login-signup"
          >
            <a
              class="btn btn-primary "
              href="/login"
            >
              Login
            </a>
            <a
              class="btn btn-warning "
              href="/register"
            >
              Sign Up
            </a>
          </div>
        </div>
      </nav>
      <div
        class="about "
      >
        <div
          class="about-content"
        >
          <div
            class="text-content"
          >
            <div
              class="left-side"
            >
              <h1>
                Eventify is your go-to platform for managing and discovering upcoming events.
              </h1>
              <p>
                Whether you're a conference center looking to showcase your events or an attendee eager to explore exciting gatherings, Eventify has you covered.
              </p>
              <p>
                <strong>
                  Effortless Event Registration Seamless Event Management Enhanced Event Visibility
                </strong>
              </p>
            </div>
          </div>
          <div
            class=" "
          >
            <div
              class="right-side"
            >
              <img
                alt="Event"
                class="profile-image"
                src="transparent.png"
                width="600px"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
